apps/meteor/app/push/server/fcm.ts

```diff
@@ -1,6 +1,6 @@
+import { serverFetch as fetch, ExtendedFetchOptions } from '@rocket.chat/server-fetch';
```

Comment:
hey, could you split the type and the function imports??

Reply:
```suggestion
import { serverFetch as fetch, type ExtendedFetchOptions } from '@rocket.chat/server-fetch';
```